src/glib/dataset.rs

```diff
@@ -0,0 +1,5 @@
+#[repr(C)]
+struct _GData;
+
+pub type GData = _GData;
```

Comment:
Why does there need to be `_GData`? It's not a public type, and the generated bindings will just create `struct GData`.